### ISSUE TITLE
presentKeyboard not firing

### ISSUE THREAD

**Issue by Dfmarulanda** [slackhq/SlackTextViewController#605]
presentKeyboard is not firing when keyboard is presenting. So inputBar keeps hidden in back of the keyboard.

**Comment by lsvz** [comment]
Same here.

**Comment by ShawnBaek** [comment]
Me too

**Comment by lsvz** [comment]
@ShawnBaek Well, I changed visibility of `slk_registerNotifications` and added it to some places in my controllers, but still sometimes it does not fire.

**Comment by ShawnBaek** [comment]
@lsvz Thanks for sharing your knowledgement. I solved this problem by remove viewDidAppear methods. But I don't know why this func affect this issues.

**Comment by lsvz** [comment]
@ShawnBaek Yea, I've just seen your issue and tried removing `viewDidAppear`, but I still do have this.
It happens after I present a controller, then tap on `textInputBar`.

**Comment by ShawnBaek** [comment]
@lsvz In my experience, I also use this library after present view controller. I hope you to solve the problem.

**Comment by johndoehelloworld** [comment]
+1 @lsvz : were you able to solve the issue?

**Comment by lsvz** [comment]
@johndoehelloworld no. Currently working on it. I will post here if I find a solution.

**Comment by lsvz** [comment]
@johndoehelloworld I managed to fix adding `slk_registerNotifications` (you have to change .h file a little bit) after I dismiss the controller.

_note that I had bug while opening a controller inside SLKTextViewController (for pictures selection)_

**Comment by blender** [comment]
Any update on this? I'm glad to take a look if no one else has updates

**Comment by blender** [comment]
I solved this by adding my slackTextViewController as a childViewController of my current VC

**Comment by fwx** [comment]
Update here. Had to make `slk_registerNotifications` public and then add `self.slk_registerNotifications` in `viewDidAppear`

**Comment by tditchek** [comment]
I'm also having this issue. Tested on iPhone 6, 8 Plus, and X, all running iOS 11.0.3. 

Are there any plans to add the changes @fwx recommends to the library so I can keep using cocoapods to manage this dependency without having to fork it?

**Comment by tditchek** [comment]
@dzenbot Any updates or workarounds for this issue? It seems to be the cause of issue #615 as well. I've tried the `slk_registerNotifications` workaround @fwx suggested above, but it's not working for me. Any help would be greatly appreciated. Thanks!
